<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>740</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>740</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QMAS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resources/icon.png</normaloff>../resources/icon.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>40</width>
    <height>40</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSpinBox" name="spin_col">
    <property name="geometry">
     <rect>
      <x>669</x>
      <y>130</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_run_by_step">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>340</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>单步运行</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>475</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_generate_obstacle">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>190</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>障碍物</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_genarate_taskpoint">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>190</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>起点/终点</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_title_input">
    <property name="geometry">
     <rect>
      <x>519</x>
      <y>13</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>输入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_run_by_stage">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>340</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>单阶段运行</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_title_input_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>手动输入：</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>380</y>
      <width>210</width>
      <height>90</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_1">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>210</width>
      <height>15</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>475</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name_row">
    <property name="geometry">
     <rect>
      <x>644</x>
      <y>130</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>行</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>210</width>
      <height>10</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_run_global">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>275</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>静态规划</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_run_Dynamic">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>275</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>动态规划</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gv_map">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="interactive">
     <bool>false</bool>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>0.000000000000000</width>
      <height>0.000000000000000</height>
     </rectf>
    </property>
    <property name="dragMode">
     <enum>QGraphicsView::NoDrag</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_name_col">
    <property name="geometry">
     <rect>
      <x>715</x>
      <y>130</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>列</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_title_output">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>278</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>输出</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_generate_map">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>155</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>生成地图</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>220</y>
      <width>210</width>
      <height>15</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lab_title_input_1">
    <property name="geometry">
     <rect>
      <x>521</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>文件输入：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_row">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>130</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_openfile">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mapsize">
    <property name="geometry">
     <rect>
      <x>565</x>
      <y>130</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string extracomment="点击按钮后点击地图的点">尺寸：</string>
    </property>
   </widget>
   <widget class="Line" name="line_0">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>208</width>
       <height>38</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="lab_filename">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>文件路径+文件名</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_readme">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string extracomment="点击按钮后点击地图的点">生成点：</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>210</width>
      <height>15</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_map_bigger">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>230</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>扩大</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_map_smaller">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>230</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>缩小</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>4</y>
      <width>210</width>
      <height>15</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lab_title_output_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>调整：</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
